backend/src/routers/course.js

```diff
@@ -2,6 +2,7 @@ const express = require('express');
 const Course = require('../models/course');
 const router = new express.Router();
 const logRequest = require('../middleware/log');
+const { StatusCodes } = require('http-status-codes');
 
 const allowedCourseDept = {
```

Comment:
@anthopark the allowedCourseDept can be convert to a json file, i.e. courseDept.json, then router can import that json. That would help clean up the router.

Reply:
> @anthopark the allowedCourseDept can be convert to a json file, i.e. courseDept.json, then router can import that json. That would help clean up the router.

Gotcha I'll make that change!